### PR TITLE
docs(time-series-data-export): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-560.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesDataExport/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+Imports:
+
+```jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { TimeseriesDataExport } from '@cognite/gearbox';
+```
+we use small functional component for demostration.
```
Comment (mrtraser) [inline src/components/TimeseriesDataExport/stories/basic.story.mdx]: better rewrite:
Functional component to demonstrate basic behavior

File: src/components/TimeseriesDataExport/stories/basic.story.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      timeseriesIds={[41852231325889, 7433885982156917]}
+      granularity={'2m'}
+      defaultTimeRange={[1567321800000, 1567408200000]}
+    />
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={TimeseriesDataExportFC} />
```
Comment (mrtraser) [inline src/components/TimeseriesDataExport/stories/basic.story.mdx]: Screenshot 2019-12-23 at 10 23 00
this 2 prop comes from HOC which is redundant knowledge for the user. Could you please just add a comment below Props section that user shouldn't specify these 2 props. We will later figure out how to hide it at all

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-560.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesDataExport/stories/basic.story.mdx [hunk 3]
```diff
@@ -26,7 +26,7 @@ import 'antd/dist/antd.css';
 import React from 'react';
 import { TimeseriesDataExport } from '@cognite/gearbox';
 ```
-we use small functional component for demostration.
+We use small Functional component to demonstrate basic behavior.
```
Comment (mrtraser) [inline src/components/TimeseriesDataExport/stories/basic.story.mdx]: I meant to remove word `we`

We use small Functional component to demonstrate basic behavior -> Wrapper component to demonstrate basic behavior

File: src/components/TimeseriesDataExport/stories/basic.story.mdx [hunk 4]
```diff
@@ -59,6 +59,8 @@ const TimeseriesChartExportWrapper: React.FC<
 
 <Props of={TimeseriesDataExportFC} />
 
+Please ignore `form*`	and `wrappedComponentRef` props.
```
Comment (mrtraser) [inline src/components/TimeseriesDataExport/stories/basic.story.mdx]: Please ignore `form*`	and `wrappedComponentRef` props. ->
**Note:** Props `form` and `wrappedComponentRef` come from `Form` HOC of antd library and displays here just as a part of the properties interface. You haven't to provide them by yourself.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-560.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-560.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-560.gearboxjs.preview.cogniteapp.com

**Pull request merged**